src/modules/home/components/property/property-card.tsx

```diff
@@ -36,18 +59,18 @@ export function ApartmentCard(apartment: Accommodation) {
           modules={[EffectFade, Navigation, Pagination]}
           className="mySwiper"
         >
-          {apartment.images.map((image, index) => (
-            <SwiperSlide key={index}>
+          {images.map((image) => (
+            <SwiperSlide key={id}>
```

Comment:
are you sure that id here is unique for each SwiperSlide?

Reply:
yes, we used uuid in backend, actually I used index instead of id, but lint or husky gave warning on it

Reply:
Actually no. id here is the id of an accommodation and you're mapping several images with the same id of the same accommodation. Use the id of each image for itself, or you can use `image` (wich is a url) itself as a key.

Reply:
Please, fix this part!